### DISCUSSION TITLE
How to make the interaction of other controls highlight instead of filtering

### DISCUSSION THREAD

**Discussion by ladioussoup** [zBritva/charticulator-visual-community#18] [Q&A]
I see that the imported field will automatically generate a highlighted field. How can I make other visual objects filter the Characteristic not just by filtering but by highlighting it?

**Comment by ladioussoup** [comment]
like this
![image](https://github.com/zBritva/charticulator-visual-community/assets/160988409/8814ac58-449f-4f70-bb26-1383cad70a1c)

**Comment by ladioussoup** [comment]
I think I know now.. When other visual objects are in a filtered state, the default state of the Character is to only display the filtered items. At this time, you need to click on the visual object on the Character to become a highlighted display mode.
But this setting is also too strange...

**Reply by zBritva** [reply]
@ladioussoup When you apply the report filters (or select values on slicer) visuals on report get filtered data. If you click on visual to select datapoint (for example click on bar chart) rest of visuals highlights correspond data (if they support data point highlights)

Anyway, I will check the case. I don't excude possible bug in the visual.

**Reply by ladioussoup** [reply]
When I selected a dimension that was not present in the control to filter it, the highlight field still had the same value as the regular field. Is this a bug
